Assets/Scripts/UI/GameOverMenu/GameOver.cs

```diff
@@ -0,0 +1,17 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class GameOver : MonoBehaviour
+{
+   public void Restart()
+    {
+        SceneManager.LoadScene("TrackTest");
```

Comment:
Not a big deal but prefer loading scenes with the build index as it is slightly faster in the backend for unity